docs/UserGuide.md

```diff
@@ -84,6 +84,19 @@ Examples:
 * `deleteMember 1 ` deletes the 1st member in the member list.
 * `deleteMember 10 ` deletes the 10th member in the member list.
 
+### Edit a Member: `editMember`
+Edits the member at the specified index with the specified attributes.
+
+Format: `editMember MEMBER_INDEX [n/MEMBER_NAME] [g/GENDER] [p/PHONE_NUMBER] [e/EMAIL] [a/ADDRESS] [t/TAG]`
+
+* The index refers to the index number shown in the displayed member list.
+* The index **must be a positive integer** that is within the range of the length of the member list.
+* At least one field to edit must be provided.
```

Comment:
Should the constraints for each field be specified?

Reply:
Noted! Have included the constraints for mistake-prone fields like gender and email.